<!--
  2 failure:  :microscope: Ther..., :disappointed_rel...
  1 warning:  :mag: Can't find ...
  1 messages
  
  DangerID: danger-id-default;
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  :microscope: There are library changes, but no changes to the unit tests. That's OK as long as you're refactoring existing code, but will require an admin to merge this PR. Please see <a href="https://github.com/vijaysingh-axway/titanium_mobile/blob/Webview_target_blank/README.md#unit-tests">README.md#unit-tests</a> for docs on unit testing.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  :disappointed_relieved: `npm test` failed. See below for details.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  :mag: Can't find junit reports at `./junit.*.xml`, skipping generating JUnit Report.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  ```
> titanium-mobile@8.1.0 test /Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-10838
> npm run ios-sanity-check && grunt


> titanium-mobile@8.1.0 ios-sanity-check /Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-10838
> ./build/scons check-ios-toplevel

Running "appcJs:src:lintOnly" (appcJs) task

Running "eslint:src" (eslint) task

/Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-10838/iphone/cli/commands/_build.js
  6410:79  warning  'out' is defined but never used  no-unused-vars

/Users/build/jenkins/workspace/ium-sdk_titanium_mobile_PR-10838/iphone/cli/hooks/frameworks.js
   61:5   warning  Expected catch() or return                  promise/catch-or-return
   61:34  warning  Avoid calling back inside of a promise      promise/no-callback-in-promise
  428:5   warning  Each then() should return a value or throw  promise/always-return

✖ 4 problems (0 errors, 4 warnings)


Running "checkFormat:ios" (checkFormat) task
Fatal error: Error: Formatting incorrect on "iphone/Classes/TiUIWebView.m", proposed changes: <?xml version='1.0'?>
<replacements xml:space='preserve' incomplete_format='false'>
<replacement offset='44858' length='6'>&#10;&#10;  </replacement>
</replacements>

npm ERR! Test failed.  See above for more details.

```
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 6d430c5d9e08038acde756e1982a67a67a1bc0f0
</p>

--- a/iphone/Classes/TiUIWebView.m
+++ b/iphone/Classes/TiUIWebView.m
@@ -1071,7 +1071,7 @@ static NSString *const baseInjectScript = @"Ti._hexish=function(a){var r='';var 
   if (!navigationAction.targetFrame.isMainFrame) {
     [webView loadRequest:navigationAction.request];
   }
-  
+
   return nil;
 }
 #pragma mark Internal Utilities

